src/_typography.scss

```diff
@@ -1,14 +1,14 @@
-:root {
-  --font-family: "Poppins", sans-serif;
-}
+@use "./colors";
+
+$font-family: "Poppins", sans-serif;
 
 h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: var(--color-heading);
+  color: map-get(colors.$color, "heading");
```

Comment:
When you do maps, there's no need in namespaces. Don't be dumb to repeat the same mistake for 1234598023495 times. What else could you pull from `colors` if not `color`? It should be `map-get(colors, "heading")`. Same for all maps